lib/auth/signInWithGithub.ts

```diff
@@ -19,5 +19,6 @@ export default async function signInWithGithub() {
     return;
   }
 
+  console.log(data.url);
```

Comment:
I don't think this is needed

Reply:
My b - didn't mean to leave that here :)